docs/docs/integrations.md

```diff
@@ -2,23 +2,42 @@
-| [Async Http Client](https://github.com/AsyncHttpClient/async-http-client)     | 8+          | 0.18+             | ✅ Nonblocking | `http4s-async-http-client` | ❌                   | ❌                | ❌              | ✅     |
-| [Ember](https://github.com/http4s/http4s)                                     | 8+          | 0.21+             | ✅ Nonblocking | `http4s-ember-client`      | `http4s-ember-server` | ✅                  | ✅                | ❌                    |
-| [Jdk11 Http Client](https://jdk-http-client.http4s.org/stable/)               | 11+         | 0.21+             | ✅ Nonblocking | `http4s-jdk-http-client`   | ❌                   | ✅ | ❌              | ✅     |
-| [Jetty](https://www.eclipse.org/jetty/)                                       | 8+          | All (0.2+)        | ?                              | ❌                        | `http4s-jetty`        |                    |                  | ✅     |
-| [OkHttp](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/) | 8+          | 0.18+             | ❌ Blocking                   | `http4s-okhttp-client`     | ❌                   | ❌                | ❌              | ✅     |
+There are also integrations for entities, metrics, and more. Check out the "Related Projects" section of the navigation menu for the complete list of integrations.
+
+## Ember
+
+Http4s Ember is a server and client backend developed in the core repository.
+
+- Implements HTTP/1 and HTTP/2
+- Runs on JDK 8+, Node.js 14+, and Scala Native
+- Pure FP, built with Cats Effect and FS2
+
+```scala
+libraryDependencies ++= Seq(
+  "org.http4s" %% "http4s-ember-server" % "@VERSION@",
+  "org.http4s" %% "http4s-ember-client" % "@VERSION@",
+)
+```
+
+## Backend Integrations
+
+| Backend                                                          | Platform                    | Http Client | Http Server | Websocket Client | Websocket Server | Proxy support (Client) |
+|------------------------------------------------------------------|-------------------------------|:-----------:|:-----------:|:----------------:|:----------------:|:----------------------:|
+| [Ember](#ember)                                                  | JDK 8+ / Node.js 14+ / Native | ✅           | ✅           | ❌                | ✅                | ❌                      |
+| [Blaze](https://github.com/http4s/blaze)                         | JDK 8+                        | ✅           | ✅           | ❌                | ✅                | ❌                      |
+| [JDK Http Client](https://jdk-http-client.http4s.org/stable/) | JDK 11+                       | ✅           | ❌           | ✅                | ❌                | ✅                      |
+| [Jetty](https://github.com/http4s/http4s-jetty)                  | JDK 8+                        | ❌           | ✅           | ❌                | ❌                | ✅                      |
```

Comment:
Maybe we should replace Jetty with Servlet. At least that one has a README.
https://github.com/http4s/http4s-servlet